x-pack/packages/ml/anomaly_utils/get_severity_types.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { type MlSeverityType, ML_ANOMALY_SEVERITY } from './anomaly_severity';
+
+let severityTypes: Record<string, MlSeverityType>;
```

Comment:
Copied this out of `anomaly_utils.ts`, I'm not sure this pattern of initializing these types is still necessary, was the idea to have some memoization?

Reply:
Refactored in [bdee7cc](https://github.com/elastic/kibana/pull/155697/commits/bdee7cc61b733494dc9c864b9ad7484ac9511e2b).